apps/api.namerank.io/pyproject.toml

```diff
@@ -0,0 +1,61 @@
+[tool.poetry]
+name = "namerank"
+version = "0.1.3"
+description = "NameHash NameRank API"
+authors = ["NameHash Team <devops@namehash.io>"]
+maintainers = ["NameHash Team <devops@namehash.io>"]
+homepage = "https://github.com/namehash/namerank"
+repository = "https://github.com/namehash/namerank"
+readme = "README.md"
+license = "LICENSE"
+packages = [{ include = "namerank" }]
+
+
+[tool.poetry.dependencies]
+python = ">=3.10,<3.13"
+ens-label-inspector = "^0.2.14"
+nameguard = "^0.1.11"
+fastapi = "^0.115.4"
+mangum = {version = "^0.17.0"}
+httpx = "^0.25.0"
+python-dotenv = "^1.0.0"
+pyahocorasick = "^2.0.0"
+setuptools = "^75.1.0"
+
+
+[tool.poetry.extras]
+namerank = ["mangum"]
```

Comment:
In nameguard this extra is called "lambda". Shouldn't it be called the same here? So you would do `pip install namerank[lambda]` instead of `pip install namerank[namerank]`.